testdata/generate.sh

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env bash
+
```

Comment:
please add `set -e`

Reply:
You wrote `test -e`.

My standard preambles are the following (depending on a situation):

```shell
#!/bin/sh

set -eu # Strict shell (w/o -o pipefail)
```

```shell
#!/bin/sh

set -xeu # Strict shell (w/o -o pipefail)
```

```shell
#!/bin/bash

set -xeuo pipefail # Strict shell
```

Motivated by http://redsymbol.net/articles/unofficial-bash-strict-mode/

I don't like playing with `IFS`, however. It feels dangerous.

Reply:
Thank you. I chose the last one. Updated.